README.md

```diff
@@ -151,6 +151,18 @@ Usage: mix coveralls.post <Options>
     -c (--committer)    Committer name ('COMMITTER' column at coveralls.io page)
     -m (--message)      Commit message ('COMMIT' column at coveralls.io page)
     -s (--sha)          Commit SHA (required when not using Travis)
+    --build             Service number ('BUILDS' column at coveralls.io page)
+    --parallel          coveralls.io 'parallel' option (See coveralls.io API Reference)
+    --subdir            Git repo sub directory: This will be added to the the front of file path, use if your covered
+                        file paths reside within a subfolder of the git repo. Example: If your source file path is
+                        "test.rb", and your git repo root is one directory up making the file's relative path
```

Comment:
I have a dummy question, so you use `"test.rb"`with `*.rb` ruby extension, it's intentional?

Reply:
Thank you for pointing this out, I took these from `coveralls.io` document, and they explained it using ruby examples. I did not modify it (it's shown [here](https://github.com/hirotnk/excoveralls_post_sample)). Maybe I'll change it to elixir file examples to reduce possible confusions.

Reply:
Yes, a good idea to change every recurrency about other extension files and replace for `ex`/`exs`

Reply:
done.